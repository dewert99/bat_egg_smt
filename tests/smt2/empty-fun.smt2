(declare-fun f (Bool) Bool)
(declare-fun g (Bool Bool) Bool)
(check-sat)
(get-model)