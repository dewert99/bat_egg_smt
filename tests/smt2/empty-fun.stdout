Sat
(
 (define-fun f ((x!0 Bool)) Bool
   f!default)
 (define-fun g ((x!0 Bool) (x!1 Bool)) Bool
   g!default)
)
